apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -71,6 +72,41 @@ const CurriculumCatalog = ({
     setExpandedCardKey(expandedCardKey === key ? null : key);
   };
 
+  // Get the top recommended similar curriculum based on the curriculum with the given
+  // curriculumKey
+  const getRecommendedSimilarCurriculum = curriculumKey => {
+    let recommendableCurricula = [...curriculaData];
+
+    // Get current curricula the others are being compared against and filter it out of the
+    // recommendable curriculum
+    const currCurriculumIndex = recommendableCurricula.findIndex(
+      e => e.key === curriculumKey
+    );
+    const currCurriculum = recommendableCurricula.splice(
+      currCurriculumIndex,
+      1
+    )[0];
+
+    // Filter out curricula that don't support any of the same grade levels
```

Comment:
Should this not be done in the `getSimilarRecommendations` function?

Reply:
This is the part of the black box definition of the recommender that I'm the least confident about. Since the recommender takes in an array of curricula to consider (so that it's possible to pass in test curricula), I figured it might make more sense to have the black box defined as passing in an array of curricula to consider and returning an array of the same length with those curricula ranked by how recommended they are. Plus, then the recommender takes fewer parameters. However, this was mostly an arbitrary decision and I'm very open to pushback if you think it makes more sense to have the filtering + ranking all encolsed within the recommender!

Reply:
In thinking a bit more, I'm swayed towards including the filtering in the black box so I'll update this pronto!